internal/character/natasha/attack.go

```diff
@@ -0,0 +1,30 @@
+package natasha
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+var attackHits = []float64{1.0}
```

Comment:
not necessary since single hit (can also omit the `HitRatio` field, will default to 1.0)